src/test/java/api/loans/anonymization/AnonymizeLoansNeverTests.java

```diff
@@ -0,0 +1,391 @@
+   *     interval after the fee/fine closes has passed
+   *     Then anonymize the loan
+   */
+  @Test
+  public void testClosedLoanWithFeesAndFinesNeverAnonymizedWhenXIntervalPassed()
+      throws InterruptedException, ExecutionException, TimeoutException, MalformedURLException {
+
+    LoanHistoryConfigurationBuilder loanHistoryConfig =
+        new LoanHistoryConfigurationBuilder()
+            .loanCloseAnonymizeNever()
+            .feeFineCloseAnonymizeAfterXInterval(20, "minute");
+    ConfigRecordBuilder config = new ConfigRecordBuilder("LOAN_HISTORY", "loan_history", loanHistoryConfig.create()
+        .encodePrettily());
+    configClient.create(config);
+    //*********************************************************************
+    IndividualResource loanResource = loansFixture.checkOutByBarcode(new CheckOutByBarcodeRequestBuilder().forItem(item1)
+        .to(user)
+        .at(servicePoint.getId()));
+    UUID loanID = loanResource.getId();
+
+    createClosedAccountWithFeeFines(loanResource, DateTime.now().minusMinutes(1));
+    loansFixture.checkInByBarcode(item1);
+
+    DateTimeUtils.setCurrentMillisOffset(20*ONE_MINUTE_AND_ONE);
+    anonymizeLoansInTenant();
+    //*********************************************************************
+    assertThat(loansStorageClient.getById(loanID).getJson(),
+        isAnonymized());
+  }
+
+  @Test
+  public void testClosedLoansWithoutFeesAndFinesNeverAnonymized()
+      throws InterruptedException, ExecutionException, TimeoutException, MalformedURLException {
+
+    LoanHistoryConfigurationBuilder loanHistoryConfig =
+        new LoanHistoryConfigurationBuilder()
+        .loanCloseAnonymizeNever()
+        .feeFineCloseAnonymizeImmediately();
+    ConfigRecordBuilder config = new ConfigRecordBuilder("LOAN_HISTORY", "loan_history", loanHistoryConfig.create()
+        .encodePrettily());
+    configClient.create(config);
+    //*********************************************************************
+    IndividualResource loanResource = loansFixture.checkOutByBarcode(new CheckOutByBarcodeRequestBuilder().forItem(item1)
+        .to(user)
+        .at(servicePoint.getId()));
+    UUID loanID = loanResource.getId();
+
+    loansFixture.checkInByBarcode(item1);
+
+    anonymizeLoansInTenant();
+    //*********************************************************************
+    assertThat(loansStorageClient.getById(loanID).getJson(),
+        not(isAnonymized()));
+  }
+
+
+  @Test
+  public void testClosedLoansWithoutFeesAndFinesNeverAnonymized2()
+      throws InterruptedException, ExecutionException, TimeoutException, MalformedURLException {
+
+    LoanHistoryConfigurationBuilder loanHistoryConfig =
+        new LoanHistoryConfigurationBuilder()
+        .loanCloseAnonymizeNever()
+        .feeFineCloseAnonymizeNever();
+    ConfigRecordBuilder config = new ConfigRecordBuilder("LOAN_HISTORY", "loan_history", loanHistoryConfig.create()
+        .encodePrettily());
```

Comment:
Can this be extracted into a method so the intent is better described and the configuration keys are not repeated in each test

Reply:
done